styleguide.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     text: 'Fork me on GitHub',
   },
   context: {
+    exampleData: path.join(__dirname, 'data/exampleData'),
```

Comment:
ExampleData

Reply:
I follow same naming convention as the current one: 'treeData' & 'topicData'